utils/src/main/java/org/apache/ode/utils/xsl/XslTransformHandler.java

```diff
@@ -165,6 +177,10 @@ public Object transform(QName processQName, URI uri, Source source,
       throw new XslTransformException(e);
     } catch (TransformerException e) {
       throw new XslTransformException("XSL Transformation failed!", e);
+    } catch (SAXException e) {
+      throw new XslTransformException(e);
```

Comment:
Would be better to add an error message too, to indicate what really went wrong.